### ISSUE TITLE
Adding this package makes the compilation process slower

### ISSUE THREAD

**Issue by tal-rofe** [TypeStrong/fork-ts-checker-webpack-plugin#740]
## Current behavior
I installed the package, found out the compilation process was slower

## Expected behavior

Should be faster

## Steps to reproduce the issue

I added the plugin to my webpack configuration:
```js
import path from 'path';

import webpack from 'webpack';
import nodeExternals from 'webpack-node-externals';
import TsconfigPathsPlugin from 'tsconfig-paths-webpack-plugin';
import ForkTsCheckerWebpackPlugin from 'fork-ts-checker-webpack-plugin';

import { version } from './package.json';

const configuration: webpack.Configuration = {
	context: __dirname,
	mode: 'production',
	target: 'node',
	entry: './src/index.ts',
	// * https://stackoverflow.com/questions/48673408/should-javascript-npm-packages-be-minified
	optimization: {
		minimize: false,
	},
	externals: [
		nodeExternals({
			modulesDir: path.resolve(__dirname, 'node_modules'),
		}),
	],
	module: {
		rules: [
			{
				test: /\.ts$/,
				use: [
					{
						loader: 'ts-loader',
						options: {
							configFile: 'tsconfig.build.json',
						},
					},
				],
				exclude: /node_modules/,
			},
		],
	},
	plugins: [
		new webpack.DefinePlugin({
			__PACKAGE_VERSION__: `'v${version}'`,
		}),
		new ForkTsCheckerWebpackPlugin({
			typescript: { configFile: 'tsconfig.build.json', context: __dirname },
		}),
	],
	resolve: {
		extensions: ['.ts'],
		plugins: [
			new TsconfigPathsPlugin({
				configFile: 'tsconfig.build.json',
				extensions: ['.ts'],
			}),
		],
	},
	output: {
		filename: 'index.js',
		path: path.resolve(__dirname, 'dist'),
	},
};

export default configuration;
```

## Issue reproduction repository

https://github.com/Vinyl-Depository/Inflint/blob/main/webpack.config.ts
## Environment
 - **fork-ts-checker-webpack-plugin**: 7.2.9
 - **typescript**: 4.6.4
 - **eslint**: 8.14.0
 - **webpack**: 5.72.0
 - **os**:  WINDOWS 10 19044.1645

**Comment by piotr-oles** [comment]
Could you elaborate on which exact command you run to compare how long it takes?
Also, please provide timings, so I can compare this with running this on my computer :)

**Comment by tal-rofe** [comment]
> Could you elaborate on which exact command you run to compare how long it takes? Also, please provide timings, so I can compare this with running this on my computer :)

I run `webpack -c webpack.config.ts`.
The timings are provided out of the box. Yes, I'll provide my timings later. Thanks

Also, I run the command of course in both cases:
1. With the plugin. Run time: `webpack 5.72.0 compiled successfully in 4838 ms`
2. W/O the plugin. Run time: `webpack 5.72.0 compiled successfully in 3214 ms`

**Comment by idanka322** [comment]
I have the same issue actually, i am not sure if i am doing something wrong or what, but when i add this plugin the compile time for dev is 21+ seconds, and without it is 2+ seconds and less than 5, im new to webpack and TS, please let me know should you need any more information.

**Comment by piotr-oles** [comment]
@tal-rofe which version of ts-loader do you use? If it's < 9.3.0, you should provide `transpileOnly: true` option

**Comment by idanka322** [comment]
@piotr-oles i am using babel-loader, and my build became too slow when using this plugin

**Comment by piotr-oles** [comment]
I wasn't able to reproduce the issue, please create new one if the problem still exists.
